src/rewriter/mkrewrites.py

```diff
@@ -147,6 +187,9 @@ def gen_mk_skolem(name, sort):
         sort_code = 'nm->mkAbstractType(kind::BITVECTOR_TYPE)'
     elif sort.base == BaseSort.AbsAbs:
         sort_code = 'nm->mkAbstractType(kind::ABSTRACT_TYPE)'
+    elif sort.base == BaseSort.BitVec:
+        # This disallows having variable BitVec sizes.
```

Comment:
```suggestion
        # This will result in a compilation error for variable BitVec sizes.
```

Reply:
I think this is the intention of this comment.

Reply:
done